assets/base.css

```diff
@@ -1170,24 +1175,45 @@ deferred-media {
 .button,
 .shopify-challenge__button,
 .customer button {
```

Comment:
Some buttons that have a transparent background are looking a little funny: 
- [account page button](https://screenshot.click/21-11-gq1p1-hscbi.png)
- [facet filtering](https://screenshot.click/21-11-bqha2-kz3gn.png)

Reply:
Good catch - I will fix them in a follow-up.  I changed the way outline buttons and inputs work in some spots but there are a few that I missed, it seems.  They should just get the background colour of their parent element rather than having a transparent background.